<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportFromSailXMLDialog</class>
 <widget class="QDialog" name="ImportFromSailXMLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Импорт из ИС &quot;Парус. Прогресс&quot;</string>
  </property>
  <widget class="QTextBrowser" name="log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>481</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnImport">
      <property name="text">
       <string>начать импортирование</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="stat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="prbImport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="wControl" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QComboBox" name="cbNet">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSelectFileXML">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Не задана</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Амбулатория</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Стационар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Скорая помощь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мобильная станция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Приемное отделение стационара</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbNet">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сеть по умолчанию:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип по умолчанию:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPersonImport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Импортировать персонал</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFileNameXML">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
